### PR TITLE
bug fix 821 Resized  list view

### PR DESCRIPTION
**Pull request by yulkats**

### CODE REVIEW

File: src/app/component/eco-news/eco-news.component.css [hunk 1]
```diff
@@ -18,3 +18,7 @@
   width: 100%;
 }
 
+.container {
+  padding-right: 0;
+  padding-left: 0;
+}
```
Comment (mplesha) [inline src/app/component/eco-news/eco-news.component.css:24]: Fix this 'No new line at EOF'

**Comment by mplesha** [comment]
conflicted PR

File: src/app/component/eco-news/eco-news.component.css (hunk 1 above)
Comment (mplesha) [inline src/app/component/eco-news/eco-news.component.css:24]: Not fixed still

Comment (yulkats) [reply src/app/component/eco-news/eco-news.component.css]: fixed

**Pull request merged**